parm/config/gfs/config.ocn

```diff
@@ -24,6 +24,11 @@ else
   export ODA_INCUPD_NHOURS="3.0"
 fi
 
-
-
+MOM6_INTERP_ICS=@MOM6_INTERP_ICS@
+if [[ "${MOM6_INTERP_ICS}" == "YES" ]]; then
+    export MOM6_RESTART_SETTING='n'
+    export MOM6_WARMSTART_FILE="MOM.res.nc"
+    export MOM6_INIT_FROM_Z='False'
+    export MOM6_INIT_UV='file'
+fi
```

Comment:
Don't we need a similar block in `gefs/config.ocn`?

Reply:
Or maybe these should be set elsewhere in the run scripts based on `$MOM6_INTERP_ICS`

Reply:
gefs/config.ocn is currently a soft link from gfs/config.ocn. i.e., each is set. If preferred to be set someplace else, I'm happy to edit

Reply:
Okay, I don't always remember which gefs files are separate and which are just links.